content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -0,0 +1,149 @@
+namespace name when creating the Pod, it will not be visible in the Kubernetes API and can only
+be discovered by tooling that has access to the affected host(s).
+
+If a static Pod fails admission control, the kubelet won't register the Pod with the
+API server. However, the Pod still runs on the node. For more information, refer to
+[kubeadm issue #1541](https://github.com/kubernetes/kubeadm/issues/1541#issuecomment-487331701)). 
+
+### Mitigations {#static-pods-mitigations}
+
+- Only [enable the kubelet static Pod manifest functionality](/docs/tasks/configure-pod-container/static-pod/#static-pod-creation)
+   if required by the node.
+- If a node uses the static Pod functionality, restrict access to the static Pod manifest directory
+   or URL to users who need the access.
+- Restrict access to kubelet configuration parameters and files to prevent an attacker setting
+   a static Pod path or URL.
+- Regularly audit and centrally report all access to directories or web storage locations that host
+   static Pod manifests and kubelet configuration files.
+
+## The kubelet API {#kubelet-api}
+
+The kubelet provides an HTTP API that is typically exposed on TCP port 10250 on cluster
+worker nodes. The API might also be exposed on control plane nodes depending on the Kubernetes
+distribution in use. Direct access to the API allows for disclosure of information about
+the pods running on a node, the logs from those pods, and execution of commands in
+every container running on the node.
+
+When Kubernetes cluster users have RBAC access to `Node` object sub-resources, that access
+serves as authorization to interact with the kubelet API. The exact access depends on
+which sub-resource access has been granted, as detailed in [kubelet authorization](https://kubernetes.io/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization).
+
+Direct access to the kubelet API is not subject to admission control and is not logged
+by Kubernetes audit logging. An attacker with direct access to this API may be able to
+bypass controls that detect or prevent certain actions.
+
+The kubelet API can be configured to authenticate requests in a number of ways. 
+By default, the kubelet configuration allows anonymous access. Most Kubernetes providers
+change the default to use webhook and certificate authentication. This lets the control plane
+ensure that the caller is authorized to access the `nodes` API resource or sub-resources.
+The default anonymous access doesn't make this assertion with the control plane.
+
+### Mitigations
+
+- Restrict access to sub-resources of the `nodes` API object using mechanisms such as
+   [RBAC](/docs/reference/access-authn-authz/rbac/). Only grant this access when required,
+   such as by monitoring services.
+- Restrict access to the kubelet port. Only allow specified and trusted IP address
+   ranges to access the port.
```

Comment:
also disable the http readonly kubelet port?

Reply:
yeah I can see it defaults to [ 0 since 1.21](https://github.com/kubernetes/kubernetes/pull/100335) (I (incorrectly) had thought it had been removed) I'll add wording around "not enabling the Kubelet read-onlyport"